curriculum/challenges/english/01-responsive-web-design/css-box-model/part-001.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 60a3e3396c7b40068ad6996a
+title: Part 1
+challengeType: 0
+dashedName: part-1
+---
+
+# --description--
+
+By now, you should be familiar with the basic elements an HTML page should have.
+
+Set up your code with a `DOCTYPE` declaration, an `html` element, a `head` element, and a `body` element.
+
+# --hints--
+
+Your code should have a `<!DOCTYPE html>` declaration.
+
+```js
+assert(code.match(/<!DOCTYPE html>/i));
+```
+
+Your code should have an `html` element.
+
+```js
+assert(document.querySelectorAll('html').length === 1);
+```
+
+Your code should have a `head` element within the `html` element.
+
+```js
+assert(document.querySelectorAll('head').length === 1);
+```
+
+Your code should have a `body` element within the `html` element.
+
+```js
+assert(document.querySelectorAll('body').length === 1);
+```
+
+Your `head` element should come before your `body` element.
+
+```js
+assert(document.querySelector('body').previousElementSibling.tagName === 'HEAD');
+```
```

Comment:
This confuses again the heck out of me, step can be passed just with adding `<!DOCTYPE html>`

Reply:
The reason is that the browser/preview automatically add those elements - and the tests are just checking that they're there.